pallets/runtime/tests/src/utility_test.rs

```diff
@@ -308,15 +307,22 @@ fn batch_secondary_with_permissions() {
         )),
     ];
     assert_ok!(Utility::batch(bob_origin.clone(), calls.clone()));
-    assert_event_doesnt_exist!(EventTest::pallet_utility(Event::BatchCompleted));
-    assert_event_exists!(EventTest::pallet_utility(Event::BatchInterrupted(0, _)));
+    assert_event_doesnt_exist!(EventTest::pallet_utility(Event::BatchCompleted(_)));
+    assert_event_exists!(
+        EventTest::pallet_utility(Event::BatchInterrupted(events, err)),
+        events.len() == 1 && events[0] == 0 && err.0 == 0
+    );
     check_name(low_risk_name);
 
     // Call the same extrinsics optimistically.
     assert_ok!(Utility::batch_optimistic(bob_origin, calls));
     assert_event_exists!(
-        EventTest::pallet_utility(Event::BatchOptimisticFailed(errors)),
-        errors.len() == 1 && errors[0].0 == 0
+        EventTest::pallet_utility(Event::BatchOptimisticFailed(events, errors)),
+        events.len() == 2
+            && events[0] == 0
+            && events[1] == 1
```

Comment:
```suggestion
        events == [0, 1]
```

Reply:
I had those on one line, but `cargo fmt` split it over multiple.  I hope it will except that.

Reply:
That is cool.  I didn't know that a vec could be compared to a slice like that.

Reply:
Yeah! This impl specifically, https://doc.rust-lang.org/nightly/std/cmp/trait.PartialEq.html#impl-PartialEq%3C%26%27_%20%5BU%5D%3E-1